cluster/charts/crossplane/templates/stack-manager-deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
         - {{ .Values.templateStacks.controllerImage | quote }}
         {{- end }}
         {{- end }}
+        {{- if .Values.forceImagePullPolicy }}
```

Comment:
this tests for non empty string values?

Reply:
Yes - the pattern is repeated from the previous block: 
`{{- if .Values.templateStacks.controllerImage }}`